master/usage/encrypt-comms.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Configuring encryption and authentication
+canonical_url: 'https://docs.projectcalico.org/master/usage/encrypt-comms'
+---
+
+## etcd connections
+
+If you are using the etcd datastore, we recommend enabling mutual TLS authentication on
+its connections as follows.
+
+- [Configure etcd](https://coreos.com/etcd/docs/latest/op-guide/security.html) to encrypt its
+  communications with TLS and require clients to present certificates signed by the etcd certificate
+  authority.
+
+- Configure each {{site.prodname}} component to verify the etcd server's identity and to present
+  a certificate to the etcd server that is signed by the etcd certificate authority.
+  - [{{site.nodecontainer}}](../reference/node/configuration)
+  - [`calicoctl`](./calicoctl/configure/etcd)
+  - [CNI plugin](../reference/cni-plugin/configuration#etcd-location)
+  - [Felix](../reference/felix/configuration#etcdv3-datastore-configuration) (on [bare metal hosts](../getting-started/bare-metal/installation/))
+
+### kube-apiserver communications (Kubernetes and OpenShift)
+
+All communications with kube-apiserver occur over TLS 1.2 with client authentication by
+default. The {{site.prodname}} components authenticate to kube-apiserver with either an x.509 certificate
+or a [JSON web token (JWT)](https://jwt.io/). You do not need to take action to secure these
```

Comment:
"You do not need to take action..." is not strictly correct - not all Kubernetes clusters are configured to provide this (although the vast majority are).

Users should make sure they have a Kubernetes cluster configured to do this.